src/server/app.js

```diff
@@ -1,13 +1,36 @@
-var express = require('express');
-var app = express();
+/* 
 
+	app.js
+
+	Initializes server. Sets up static content. Establishes routes and 
+	their handlers.
+
+	Written by Sam Reaves
+	July 22, 2015
+
+ */
+
+
+// Import Express and initialize server. 
+var express = require('express'),
+	app = module.exports.app = exports.app = express(),
+	path = require('path');
+
+// Establish static directory at client/public
+app.use(express.static(path.join(__dirname, '../client/static')));
+
+// Route handler for root - sends current static landing page
 app.get('/', function(req, res) {
-  res.send('Hello, World');
+  res.sendfile("index.html");
 });
 
+// Server starts listening on port 3000.
 var server = app.listen(3000, function() {
```

Comment:
At some point soon we should move this port number out of here, at the very least to a CAPS_LOCKED var near the top of the file (that we will actually call a ```const``` instead of a ```var``` once we get es6 transpiler working).

However, for this particular variable, it would actually be really nice to accept a command line argument. So maybe one of your next tasks can be to integrate this: https://github.com/substack/node-optimist and use it to take an optional *port* parameter in so it would be ```npm start 3000```